Dockerfile

```diff
@@ -3,17 +3,19 @@
 # ------------------------------------------------------------------------------
 FROM ruby:3.1.0-alpine as base
 
-RUN apk add --no-cache --no-progress \
-    build-base less curl tzdata postgresql-dev yarn gcompat \
+RUN apk add --no-cache --no-progress build-base less curl tzdata gcompat \
     "gmp>=6.2.1-r1" "zlib>=1.2.12-r0" "busybox>=1.34.1-r5" \
     "libretls>=3.3.4-r3" "libssl1.1>=1.1.1n-r0" "libcrypto1.1>=1.1.1n-r0"
 
 # ------------------------------------------------------------------------------
-# Production Stage
+# Production Stage - nodejs v16.14.2, postgresql v13.6
```

Comment:
Do we really want to be hard coding these version numbers here? This looks like the sort of thing that won't be maintained, will rapidly fall out of date, and will then just be misleading.